src/main/java/racingcar/model/Register.java

```diff
@@ -0,0 +1,49 @@
+package racingcar.model;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+public class Register {
+    public ArrayList<Car> carList;
+
+    public Register() {
+        this.carList = new ArrayList<>();
+    }
+
+    public void carListUp(ArrayList<String> participateCars) {
+        duplicationValidation(participateCars);
+        for (String name : participateCars) {
+            carList.add(new Car(name));
+        }
+    }
+
+    public void duplicationValidation(ArrayList<String> duplicateCheck) {
```

Comment:
```suggestion

    public void duplicationValidation(ArrayList<String> names) {
```

파라미터명이 더 구체적이면 좋겠습니다!!